word_guess.rb

```diff
@@ -0,0 +1,110 @@
+require "colorize"
+
+# class Image generates beating heart
+class Image
+  attr_writer :num_attempts
+  def initialize
+    @num_attempts = 5
+    @heart_parts = ["  ,d88b.d88b,  ", "  88888888888  ", "  `Y8888888Y'  ", "    `Y888Y'    ", "      `Y'      "]
```

Comment:
This line is a little hard to understand as is - you could make it more readable by moving things around:

```ruby
@heart_parts = [
  "  ,d88b.d88b,  ",
  "  88888888888  ",
  "  `Y8888888Y'  ",
  "    `Y888Y'    ",
  "      `Y'      "
]